guides/common/modules/con_host-substatus-overview.adoc

```diff
@@ -1,55 +1,247 @@
 [id="host-substatus-overview_{context}"]
 = Host Sub-status Overview
 
-A sub-status monitors only part of a host's capabilities.
+A sub-status monitors only a part of host's capabilities.
 
-Currently, {Project} ships only with *Build* and *Configuration* sub-statuses.
-There can be more sub-statuses depending on which plugins you add to your {Project}.
+{Project} contains the following sub-statuses:
 
-The *build* sub-status is relevant for managed hosts.
+Configuration::
+This sub-status is only relevant if {Project} uses a configuration management system like Ansible, Puppet, or Salt.
++
+Possible values:
+// TODO: numbers
++
+[options="header"]
+|===
+| Value | Global Host Status | Number
+| Active | OK | 0
+| Pending | OK | 1
+| No changes | OK | 2
+| No reports | OK / Warning | 3
+| Out of sync | Warning | 4
+| Error | Error | 5
```

Comment:
I don't think there are numbers. At most there are bit numbers (since the total value is a number and things are calculated bitwise.

You probably convert https://github.com/theforeman/foreman/blob/b5e5dfb4108d36210a1691af45d6d8b980f4f6d1/app/models/host_status/configuration_status.rb#L43-L59

reading this, wouldn't it make more sense to flip it and start with the global status and explain what may be the case?

Reply:
Edit: I misunderstood the comment. Removing numbers from this sub-status.